advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+
+![pgAdmin](Images/pgAdmintargettables.png)
+
+## Data Transformation
+
+Precisely Connect CDC has options that let you modify the source data you are distributing or even create or derive the data you distribute. You can use predefined functions to accomplish these data transformations or you can construct your own operations or functions.
+
+For example, to express source currency values in terms of a different currency, you specify the transformation algorithm to use for that column mapping. For some common transformations, the algorithm is supplied as a predefined Connect CDC function.
+
+The algorithms you specify are known in the Connect CDC Director as expressions. The Connect CDC Expression Handler parses them and then creates runtime code that calculates the result for the expressions.
+
+In addition to or instead of calls to predefined functions, expressions may have an arbitrary number of constants, column references, and calculations on those constants and columns and functions. Their results must be compatible in type and length with the target column.
+
+### Steps to demonstrate data transformations
+
+The following steps demonstrate data transformations.
+
+1. Open Connect CDC Director.
+
+2. Go to `Source DBMS Server` —> `Tables` —> `Sending Tables`.
+
+3. Select the table(s) to perform data transformation using the checking box in front of them and right click and select `Properties` —> `Mappings`.
+
+4. On the `Mapping` tab, you define the column mappings for each of your target tables with respect to the source tables. The `Mapping` tab displays the corresponding columns in the `Target Column` and `Source Data` columns in each row. Connect CDC Director automatically mapped target columns to source columns that have the same name.
+
+When the Connect CDC Director maps a source table to a same-named target table, it also associates individual source table columns with same-named target table columns. You then select which of these column pairings you want to include in your data distribution, which additional column pairings you want to arrange, and what type of data transformation, if any, you want to assign to each column pairing.
+
+Mappings Tab has the following options
+
+    - Receiving server list : The name of the target server is displayed.
+    - Receiving table list : The name of the target table is displayed. If it does not (for example, because its name does not match your source table’s), select the target table from the Receiving table list. The list includes all tables in the list of Available tables for the target server. Any table name with [M]preceding it is already mapped to a source table.
+    
+The Mapping tab grid displays each target table column and its datatype as well as its default corresponding source column, if any. Copy column under Method in the grid means the source column value is to be distributed without any special handling or transformation.
```

Comment:
Same with these items. They are in like a block form.

Reply:
yes its intentional.